tests/units/common/parsing/test_generic_parsing.py

```diff
@@ -189,3 +190,31 @@ def test_parse_authors(test_input, expected):
 )
 def test_parse_thesis_supervisors(test_input, expected):
     assert expected == parse_thesis_supervisors(test_input)
+
+
+def test_publication_info():
+    test_article = {
+        "journal_title": "Test Value",
+        "journal_volume": "Test Value",
+        "journal_year": "2022",
+        "journal_issue": "Test Value",
+        "journal_artid": "",
```

Comment:
could you please add a test for the happy path as well (all the values are populated)?